### DISCUSSION TITLE
Troubleshooting section in the wiki?

### DISCUSSION THREAD

**Discussion by benkoshy** [Nice3point/RevitTemplates#54] [Ideas]
@Nice3point  - Huge thanks for your library:

- I came across a few hiccups along the way when trying to get a 'hello world' up and running (I am new to the Revit API.). Perhaps it may benefit the community if it is documented under the 'trouble shooting' section of the wiki? However I am unable to edit  the wiki (probably because I do not have permission). If you permit i can add it in there? and if you  don't like it you can simply revert the changes.

**Comment by Nice3point** [comment]
Hi, you can attach the text you want to add to the issues. Or describe a 'trouble shooting' so that I can document it myself later if I think it's necessary
